backend/src/main/java/com/speakfluid/backend/usecases/TranscriptAnalysisInteractor.java

```diff
@@ -21,19 +21,19 @@
 public class TranscriptAnalysisInteractor implements TranscriptAnalysisInputBoundary{
 
     // Generate all TalkStep entities,except Card
-    ButtonStep buttonStep = new ButtonStep();
+    //ButtonStep buttonStep = new ButtonStep();
     ImageStep imageStep = new ImageStep();
     CaptureStep captureStep = new CaptureStep();
     CarouselStep carouselStep = new CarouselStep();
     TextStep textStep = new TextStep();
     ChoiceStep choiceStep = new ChoiceStep();
-    CardStep cardStep = new CardStep();
+    //CardStep cardStep = new CardStep();
```

Comment:
ButtonStep and CardStep are commented out due to a small issue in the keyword attributes.